assets/home/link.js

```diff
@@ -29,13 +17,12 @@ const getHostname = ( url ) => {
  * @param {Function} props.onClick The event listener for the click event.
  */
 const Link = ( { label, url, onClick } ) => {
-	const isExternal = getHostname( window.location ) !== getHostname( url );
-
+	const isExternal = isUrlExternal( url );
 	return (
 		<div className="sensei-home__link">
 			<a
 				href={ addUtms( url ) }
-				target={ onClick ? undefined : '_blank' }
+				target={ onClick || ! isExternal ? undefined : '_blank' }
```

Comment:
I think we could also just add the `rel="noreferrer"` when it's external, right?

Reply:
I tried to do that, but ESLint complained as if the `rel="noreferrer"` attribute wasn't defined..so there's that.

I think I'll create a separate object called `linkProps` just to handle this. :thinking:

Reply:
Added in 8d34f53cea7bdd16742730ef03ae0ba11f264085 - Thanks!

Reply:
Thank you for the tweaks!